content/src/fieldConfig/config.json

```diff
@@ -21,6 +21,9 @@
     "requiredTasksSingular": "${numTasks} required task",
     "requiredTasksPlural": "${numTasks} required tasks"
   },
+  "skipToMainContent": {
+    "buttonText": "Skip to main content"
```

Comment:
I also imagined it with more Capital letters? `Skip to Main Content`

Reply:
This is maintains the current implementation. We can ask Content, but I don't want to modify this without explicit instruction.